geolocator_windows/pubspec.yaml

```
@@ -0,0 +1,28 @@
+name: geolocator_windows
+description: Geolocation Windows plugin for Flutter. This plugin provides the Windows implementation for the geolocator.
+version: 0.1.0
+repository: https://github.com/Baseflow/flutter-geolocator/tree/master/geolocator_windows
+issue_tracker: https://github.com/Baseflow/flutter-geolocator/issues?q=is%3Aissue+is%3Aopen
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.20.0"
+
+flutter:
+  plugin:
+    platforms:
+      windows:
+        pluginClass: GeolocatorWindows
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  geolocator_platform_interface: ^4.0.0
+  
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.4
+  mockito: ^5.0.0-nullsafety.7
+  plugin_platform_interface: ^2.0.0
```

Lets make sure these are also updated to the latest versions (especially the `plugin_platform_interface` package contains an important bug fix):
```suggestion
  mockito: ^5.0.17
  plugin_platform_interface: ^2.1.2
```